KDIC/KDICScrapperReqs.swift

```diff
@@ -1,15 +1,7 @@
-//
-//  KdicScrapperReqs.swift
-//  KDIC
-//
-//  Created by Shaun Mataire on 2/11/16.
-//  Copyright © 2016 AppDev. All rights reserved.
-//
-
 import UIKit
 import Foundation
 
-class KdicScrapperReqs{
+class KDICScrapperReqs{
```

Comment:
I thought you were going to change this into a Singleton type class so it would be called like

`KDICScraper.scrape(url NSURL, completion: (articles: [Article]?, error: ErrorType?))`

This would clean up a lot of code in this class and in the `BlogViewController`